src/stdatamodels/jwst/datamodels/__init__.py

```diff
@@ -66,6 +66,7 @@
 from .saturation import SaturationModel
 from .segmap import SegmentationMapModel
 from .slit import SlitModel, SlitDataModel
+from .soss_tracemodel import SossTraceModel
```

Comment:
```suggestion
from .sosstracemodel import SossTraceModel
```
One nitpick... can we remove the `_` to keep it consistent with the other soss stuff?